src/main/scala/com/ibm/couchdb/api/Documents.scala

```diff
@@ -30,40 +30,40 @@ class Documents(client: Client, db: String, typeMapping: TypeMapping) {
   val types  = typeMapping.types
   val server = new Server(client)
 
-  private def getClassName[T](obj: T): String = obj.getClass.getCanonicalName
+  private def getClassName[S](obj: S): String = obj.getClass.getCanonicalName
```

Comment:
We should keep this type parameter name consistent with other places, what do you think about changing it everywhere to `D` (for Document)? It's already called `D` in https://github.com/beloglazov/couchdb-scala/blob/updateDependencyVersions/src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala#L107